<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>574</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ProcILLUMINATI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="ButtonDeleteWatcher">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckBoxActiveWatcher">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Активный</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="0">
       <widget class="QListWidget" name="LoggingListWidget"/>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="MemRefreshButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Старт</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignBottom">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Надсмотрщики&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Память процесса</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Лог</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Частота обновления</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="FrequencySlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" rowspan="11">
       <widget class="QTextEdit" name="MemoryText">
        <property name="minimumSize">
         <size>
          <width>289</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>12</pointsize>
          <stylestrategy>NoAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QListWidget" name="WatchersListView"/>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>60</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="ProcessMenu">
    <property name="title">
     <string>Процесс</string>
    </property>
    <addaction name="AttachToProcessAction"/>
    <addaction name="CheckMemoryAction"/>
    <addaction name="AddWatcherAction"/>
   </widget>
   <widget class="QMenu" name="LogMenu">
    <property name="title">
     <string>Лог</string>
    </property>
    <addaction name="SaveLogAction"/>
   </widget>
   <widget class="QMenu" name="menu_QT">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_Qt"/>
   </widget>
   <addaction name="ProcessMenu"/>
   <addaction name="LogMenu"/>
   <addaction name="menu_QT"/>
  </widget>
  <action name="AttachToProcessAction">
   <property name="text">
    <string>Подцепиться к процессу</string>
   </property>
  </action>
  <action name="SaveLogAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="CheckMemoryAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Посмотреть область памяти</string>
   </property>
  </action>
  <action name="AddWatcherAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Добавить надсмотрщика</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>Про Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
